src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const elements = [...document.querySelectorAll('li')];
```

Comment:
The use of the spread operator to create an array from NodeList is correct. Ensure that the 'li' elements have the necessary data attributes (position, salary, age) for the code to function properly.